tests/unit/src/test/scala/tests/codeactions/ImportMissingSymbolLspSuite.scala

```diff
@@ -458,6 +458,18 @@ class ImportMissingSymbolLspSuite
     kind = List(ImportMissingSymbolQuickFix.kind),
   )
 
+  checkActionsOnly(
+    "i7777 (<empty> package)",
```

Comment:
Looks like this is breaking the windows tests, might be good to do a simple name (directory name is derived from that)